src/Components/Blueprints/BlueprintsPagination.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  OnSetPage,
+  Pagination,
+  PaginationVariant,
+} from '@patternfly/react-core';
+
+import {
+  selectBlueprintSearchInput,
+  selectLimit,
+  selectOffset,
+  setBlueprintLimit,
+  setBlueprintsOffset,
+} from '../../store/BlueprintSlice';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { useGetBlueprintsQuery } from '../../store/imageBuilderApi';
+
+const BlueprintsPagination = () => {
+  const blueprintSearchInput = useAppSelector(selectBlueprintSearchInput);
+  const blueprintsOffset = useAppSelector(selectOffset);
+  const blueprintsLimit = useAppSelector(selectLimit);
```

Comment:
we could set the fallbacks here, unless it would break the useQuery of course
```suggestion
  const blueprintsOffset = useAppSelector(selectOffset) || 0;
  const blueprintsLimit = useAppSelector(selectLimit) || 10;
```